### PR TITLE
Bug#1517634 - Rearranged instructions and corrected formatting

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1517634
 - Rearranged the instructions in correct order as per the bug description
 - Removed old code formatting instances of `====` surrounding code blocks.
 - Added `[source, bash]` tags to relevant code blocks.
 - Changed `#` to `$` for all code blocks to fix rendering issues added by the previous change.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bfallonf Can you PTAL?

No major changes, I only rearranged some instructions and corrected formatting. No revision history needed.

File: install_config/install/disconnected_install.adoc [hunk 1]
```diff
@@ -134,8 +139,9 @@ attach it:
 yum repositories, and `createrepo` can create a usable `yum` repository from a
 directory:
 +
+[source, bash]
 ----
-# yum -y install yum-utils createrepo docker git
+$ yum -y install yum-utils createrepo docker git
```
Comment (bfallonf) [inline install_config/install/disconnected_install.adoc]: @gaurav-nelson I think this needs a `sudo` in front if it's not going to be run as root user.

File: install_config/install/disconnected_install.adoc [hunk 2]
```diff
@@ -376,8 +395,9 @@ webserver.
 If you can re-connect the server where you synchronized the software and images
 to your LAN, then you can simply install Apache on the server:
 
+[source, bash]
 ----
-# yum install httpd
+$ yum install httpd
```
Comment (bfallonf) [inline install_config/install/disconnected_install.adoc]: Same comment about `sudo` or root user

**Comment by bfallonf** [comment]
@gaurav-nelson I made a comment about the `yum` commands needing `sudo` if they're going to be run as a non-root user. Not sure if this applies to any other commands.

**Comment by gaurav-nelson** [comment]
Thank you @bfallonf 

I have added your suggestions in the latest commit.

**Comment by gaurav-nelson** [comment]
Ready to merge https://bugzilla.redhat.com/show_bug.cgi?id=1517634#c2

**Pull request merged**